packages/worker/src/tests/TestConfiguration.ts

```diff
@@ -20,22 +20,45 @@ import {
   auth,
   constants,
   env as coreEnv,
+  utils,
+  DEFAULT_TENANT_ID,
 } from "@budibase/backend-core"
-import structures, { TENANT_ID, TENANT_1, CSRF_TOKEN } from "./structures"
+import structures, { TENANT_ID, CSRF_TOKEN } from "./structures"
 import { CreateUserResponse, User, AuthToken } from "@budibase/types"
 import API from "./api"
+import sdk from "../sdk"
 
 enum Mode {
   CLOUD = "cloud",
   SELF = "self",
 }
 
+async function retry<T extends (...arg0: any[]) => any>(
```

Comment:
Similar to the `retry` function in server? 
Could be moved out to `backend-core`